<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultWindow</class>
 <widget class="QWidget" name="ResultWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>680</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="horizontalSpacing">
        <number>7</number>
       </property>
       <item row="0" column="0">
        <widget class="AudioFileSelector" name="audio_file_selector" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>10</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="JsScriptMultiSelector" name="js_script_multi_selector" native="true">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="AudioViewModeSelector" name="audio_view_mode_selector" native="true">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_execute">
         <property name="text">
          <string>Execute</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QCustomPlot" name="plot" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>8</horstretch>
         <verstretch>8</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">ui/qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AudioFileSelector</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/audio_file_selector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AudioViewModeSelector</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/audio_view_mode_selector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JsScriptMultiSelector</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/js_script_multi_selector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
